go/ql/lib/ext/runtime.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/go-all
+      extensible: summaryModel
+    data:
+      - ["runtime", "stringer", True, "String", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
```

Comment:
What's up with this? I can't find any mention of it, and `stringer` looks like it wouldn't be exported.